test/orchard/misc_test.clj

```diff
@@ -81,3 +81,7 @@
   (is (= 3 (misc/safe-count [1 2 3])))
   (is (nil? (misc/safe-count (repeat 1))))
   (is (nil? (misc/safe-count :not-a-seq))))
+
+(deftest safe-read-edn
+  (is (= nil (misc/safe-read-edn "[")))
```

Comment:
`misc/safe-read-edn`'s docstring refers to tagged literals. Surely they deserve some coverage here?

Reply:
Fixed